apps/docs/pages/docs/v2/Components/image.en-US.mdx

```diff
@@ -0,0 +1,31 @@
+---
+searchable: true
+---
+
+import { CodeEditor } from '@components/code-editor';
+
+# Image
+
+Wrapper around `Image` component from `react-native`, it accepts every props from react native `Image` component.
+
+## Import
+
+```js
+import { Image } from "@ficus-ui/native";
+```
+
+## Usage
+
+<CodeEditor code={`<Image
+  h={200}
+  w={300}
+  borderRadius={10}
+  source={{
+    uri:
+      'https://www.picturethisai.com/wiki-image/1080/153400456440184865.jpeg',
+  }}
```

Comment:
Can we imagine to add a shortcut like `src` or `uri` at first level to avoid this a bit heavy required syntax?

I want to be able to use the component that way:
```
<Image
  h={200}
  w={300}
  borderRadius={10}
  src="https://www.picturethisai.com/wiki-image/1080/153400456440184865.jpeg"
/>
```